config/tracks/road_atlanta2018.ini

```diff
@@ -1,5 +1,8 @@
- ; config was prepared automatically. source:
- ; https://github.com/ac-custom-shaders-patch/acc-extension-config/blob/master/config/tracks/road_atlanta2018.ini
+; config was prepared automatically. source:
+; https://github.com/ac-custom-shaders-patch/acc-extension-config/blob/master/config/tracks/road_atlanta2018.ini
+; config was prepared automatically. source:
+; https://github.com/ac-custom-shaders-patch/acc-extension-config/blob/master/config/tracks/road_atlanta2018.ini
+; Vannipo: New [MATERIAL_ADJUSTMENT_14], replaced all conditions NIGHT_SMOOTH, NIGHT_SMOOTH_HEATING to SMOOTH_SUN_A
 [ABOUT]
 AUTHOR = Blamer
 VERSION = 2.0.2

```

Comment:
Update version number and add notes in NOTES

Reply:
Done. Hopefully correct